libretro/libretro-core.c

```diff
@@ -507,27 +507,27 @@ void retro_set_environment(retro_environment_t cb)
 #if  defined(__VIC20__)
       {
          "vice_vic20_model",
-         "VIC20 model; VIC20MODEL_VIC20_PAL|VIC20MODEL_VIC20_NTSC|VIC20MODEL_VIC21|VIC20MODEL_UNKNOWN",
+         "VIC20 model; VIC20 PAL|VIC20 NTSC|VIC21",

```

Comment:
The model.h file shows in a comment that the vic21 model is the `SuperVIC (+16K)` can you change it? Thanks. I think the more accurate the model description are the better.